README.md

```diff
@@ -1,21 +1,243 @@
 # client-cert-update #
 
-[![Build Status](https://travis-ci.com/cisagov/client-cert-update.svg?branch=develop)](https://travis-ci.com/cisagov/client-cert-update)
-[![Total alerts](https://img.shields.io/lgtm/alerts/g/cisagov/client-cert-update.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/client-cert-update/alerts/)
-[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/cisagov/client-cert-update.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/cisagov/client-cert-update/context:python)
+[![GitHub Build Status](https://github.com/cisagov/client-cert-update/workflows/build/badge.svg)](https://github.com/cisagov/client-cert-update/actions/workflows/build.yml)
+[![CodeQL](https://github.com/cisagov/client-cert-update/workflows/CodeQL/badge.svg)](https://github.com/cisagov/client-cert-update/actions/workflows/codeql-analysis.yml)
+[![Known Vulnerabilities](https://snyk.io/test/github/cisagov/client-cert-update/badge.svg)](https://snyk.io/test/github/cisagov/client-cert-update)
+
+## Docker Image ##
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/cisagov/client-cert-update)](https://hub.docker.com/r/cisagov/client-cert-update)
+[![Docker Image Size (latest by date)](https://img.shields.io/docker/image-size/cisagov/client-cert-update)](https://hub.docker.com/r/cisagov/client-cert-update)
+[![Platforms](https://img.shields.io/badge/platforms-amd64%20%7C%20arm%2Fv6%20%7C%20arm%2Fv7%20%7C%20arm64%20%7C%20ppc64le%20%7C%20s390x-blue)](https://hub.docker.com/r/cisagov/client-cert-update/tags)
 
 This project contains code for sending FNR BOD an updated list of the
 live websites in .gov that require authentication via client
 certificates.
 
+## Running ##
+
+### Running with Docker ###
+
+To run the `cisagov/client-cert-update` image via Docker:
+
+```console
+docker run cisagov/client-cert-update:0.0.2
+```
+
+### Running with Docker Compose ###
+
+1. Create a `docker-compose.yml` file similar to the one below to use [Docker Compose](https://docs.docker.com/compose/).
+
+    ```yaml
+    ---
+    version: "3.7"
+
+    services:
+      update:
+        image: cisagov/client-cert-update:0.0.2
+        init: true
+        environment:
+          - AWS_CONFIG_FILE=path/to/aws_config
+          - AWS_PROFILE=default
+    ```
+
+1. Start the container and detach:
+
+    ```console
+    docker compose up --detach
+    ```
+
+## Using secrets with your container ##
+
+This container also supports passing sensitive values via [Docker
+secrets](https://docs.docker.com/engine/swarm/secrets/).  Passing sensitive
+values like your credentials can be more secure using secrets than using
+environment variables.  See the
+[secrets](#secrets) section below for a table of all supported secret files.
```

Comment:
Odd line breaking here.  Why not this?
```suggestion
environment variables.  See the [secrets](#secrets) section below for a table
of all supported secret files.
```

Reply:
I'm just usin' [what I was given](https://github.com/cisagov/skeleton-docker/blob/9cea64da4a1379cdde3c509a09fc548f0da8b034/README.md?plain=1#L63-L65).

Reply:
What the hell was @jsf9k thinking? 😝

Reply:
Who knows...

Reply:
Resolving this conversation.  @mcdonnnj - feel free to accept my suggestion above or not.

Reply:
@dav3r I'll take it up to [cisagov/skeleton-docker](https://github.com/cisagov/skeleton-docker).